index.css

```diff
@@ -0,0 +1,289 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+.blink {
+  color: #f8cb45;
+}
+.it {
+  color: #54b226;
+}
+.delivery {
+  height: 60px;
+  display: flex;
+  flex-direction: row;
```

Comment:
`display: flex;
  flex-direction: row;`
You have repeated above code at many places. 
You can create separate class for just that which will be called `flex-row` and you can reuse that all places